src/views/UserResView.vue

```diff
@@ -1,15 +1,14 @@
 <template>
-  <user-home />
+  <user-res />
```

Comment:
Revert, UserHome contains available reservations and the user's reservations. User-res makes it seem like it's just their reservations.